docs/src/use/command-line-interface.md

```diff
@@ -97,7 +97,7 @@ Output:
 
 Inline configuration comments:
   --no-inline-config              Prevent comments from changing config or rules
-  --report-unused-disable-directives  Adds reported errors for unused eslint-disable directives
+  --report-unused-disable-directives  Adds reported errors for unused eslint-disable and eslint-enable directives
```

Comment:
If we want to update this description (which appears in the output of `eslint --help`), we should update it here as well:

https://github.com/eslint/eslint/blob/b7ef2f34fe12b68a366e1b4bf5f64d7332c6e72e/lib/options.js#L303-L308